docs/website/mtouch-errors.md

```diff
@@ -673,6 +673,15 @@ When using --recursive-directories, only 1 assembly should match
 
 <!-- 0134 used by mmp -->
 
+### <a name="MT0135"/>MT0135: Did not link system framework '{0}' as Xcode SDK '{1}' is too old.
+
+Xamarin.iOS contains API bindings for the listed framework which is newer than the Xcode used. 
+
+They may fail at runtime as we are not linking against the required system framework.
+
+Consider updating to a newer Xcode, or if those APIs are not used, enabling [linking](https://docs.microsoft.com/en-us/xamarin/mac/deploy-test/linker).
```

Comment:
So this 0135 error is a bit similar to https://docs.microsoft.com/en-us/xamarin/ios/troubleshooting/mtouch-errors#MT0091 in the sense you want people to update their Xcode or enable the linker. I think you should copy or take inspiration from the detailed "enable the linker" explanation in 0091.

Also I have to admit that I'm a bit confused with the general wording of this error, not sure I understand.

If I go line by line:

> Did not link system framework '{0}' as Xcode SDK '{1}' is too old.

ok, couldn't, link Xcode too old.

> Xamarin.iOS contains API bindings for the listed framework which is newer than the Xcode used.

The framework is newer than the Xcode used? That sounds weird. I don't understand this.

> They may fail at runtime as we are not linking against the required system framework.

The listed framework may fail or the API bindings :P ? The error is about a system framework and it's also the last element of the previous sentence so I think that grammatically "they" applies to the listed frameworks (am I wrong?).

Overall I'm confused with those 2 middle sentences that are supposed to explain the core of the issue (maybe it's just me).

If I try really hard to parse this error I get that it's basically https://docs.microsoft.com/en-us/xamarin/ios/troubleshooting/mtouch-errors#MT0091.
We can't "link" some frameworks because they are simply not available in the currently selected Xcode (too old). Some APIs in Xamarin.iOS depend on those frameworks (hence the error) and because we couldn't link them it's gonna fail at runtime. You can either upgrade Xcode to get the new framework or enable the linker so those APIs are skipped.

Anyway, we _might_ need to rephrase this, it's not crystal clear for me on the first read, it's very technical and hard to digest and I don't know if users will just understand this.